css-ui-3/caret-color-016.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color visited link computed value</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<link rel="help" href="https://www.w3.org/TR/css3-color/#color0">
+<meta name="flags" content="dom interact">
```

Comment:
The `dom` flag isn't, wrong, but we've recently decided to deprecate it, as it can be infered from the presence of a script, so you can remove it.

@gsnedders , can you confirm?